runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangePlugin.java

```diff
@@ -674,6 +681,58 @@ public void onScriptCallbackEvent(ScriptCallbackEvent event)
 		stringStack[stringStackSize - 1] += titleBuilder.toString();
 	}
 
+	private void setLimitReset(int itemId)
+	{
+		String lastLimit = getLimitReset(itemId);
+		if (!lastLimit.equals("None"))
+		{
+			return; // do not update if previous limit has not expired
+		}
+		String itemIdStr = Integer.toString(itemId);
+		DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm");
+		LocalDateTime now = LocalDateTime.now();
+		String currentDateTime = now.format(formatter);
+		configManager.setConfiguration("gelimitreset." + client.getUsername().toLowerCase(),
+			itemIdStr, currentDateTime);
```

Comment:
Just store an `Instant` in the config instead of a formatted local date time string.

Reply:
I can do that, but just to learn, why wouldn't a formatted datetime string be good?
Also, to store an `Instant` in the config, I would have to change it into a String format right?

Reply:
The config system already supports storing `Instant`, you should just be able to give it that.

Reply:
Done!